server/routerlicious/packages/services-utils/src/logger.ts

```diff
@@ -67,3 +67,5 @@ export function configureLogging(configOrPath: nconf.Provider | string) {
         args[0] = `${name} ${args[0]}`;
     };
 }
+
+export {winston};
```

Comment:
Can you tell me a bit about what happened here?  I'm guessing 'configureLogging(..)' wasn't having an effect?